bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -164,28 +139,24 @@ Rollershutter V_DG_M_W  "Velux DG Window Mitte [%d]"    { velux="thing=bridge;ch
 sitemap velux label="Velux Environment"
 {
     Frame label="Velux Shutter and Window" {
-
         Switch  item=V_DG_W_S
         Switch  item=V_DG_O_S
         Switch  item=V_DG_M_S
         Switch  item=V_DG_M_W
-
     }
-    
     Frame label="Velux Bridge" {
         Text    item=V_CHECK
         Text    item=V_STATUS
         Text    item=V_FIRMWARE
     }
-    
 }
 ```
 
 ### Rule velux.rules
```

Comment:
This line should say just "Rules". "velux.rules" belongs above the code fence that starts the set of examples.